features/other_b/series.feature

```diff
@@ -135,6 +135,10 @@ Feature: Create and Edit Series
       And I view the series "Ponies"
     Then I should see "Sweetie Belle"
       And I should see "Starsong"
+    When I follow "Pointless Pseud"
```

Comment:
The title check looks good! However, this is already a really, really long scenario (which is totally not your fault!) so instead of making it longer (which makes it harder to figure out what's broken if it starts to fail), I'd suggest moving the page title bit to its own scenario. 

The set up for creating a series with a pseud is actually a lot less involved than this existing scenario suggest. It would just be:
```
  Scenario: A pseud's series page contains the pseud in the page title
    Given I am logged in as "author"
      And I add the pseud "Pointless Pseud"
      And I set up the draft "Sweetie Belle" as part of a series "Ponies"
      And I select "Pointless Pseud" from "work_author_attributes_ids_"
      And I press "Post"
```
followed by your When and Then steps. (You can omit the `I should see "Ponies"` and just go for checking the page title.)

If you want to go back through this "Three ways to add a work to a series: a user with more than one pseud" and try to shorten a few of the parts where it posts a work, that might be a good way to get more familiar with all the options we have for creating works in our Cucumber tests. However, it's totally optional! 

(Also, I'm done now -- sorry about the trigger-happy comment finger. ;))